docs/dev/test-configuration-options.md

```diff
@@ -557,7 +557,7 @@ Use this capability to enable animations for real devices by setting it to `true
 ### `resigningEnabled`
 <p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> |</small></p>
 
-Appium override setting that enables the resigning (iOS) or instrumentation (Android) of apps on the Sauce Labs side, allowing the usage of the other capabilities listed in this section.
+Appium override setting that enables the resigning (iOS) or instrumentation (Android) of apps on the Sauce Labs side, allowing the usage of the other capabilities listed in this section. See [Resigning Enablements](/mobile-apps/automated-testing/ipa-files/#sauce-resigning-enablements) for more information.
```

Comment:
tbh, this is not an Appium cap, when we need to adjust our docs to W3C for Appium this will fall under the `sauce:options`

Reply:
so instead of `Appium override` we could call it a custom Sauce Labs capability to denote it lives in `sauce:options`